target/linux/ar71xx/files/arch/mips/ath79/mach-rbspi.c

```diff
@@ -255,6 +255,8 @@ static struct gpio_led rb952_leds[] __initdata = {
 
 
 /* RB 962UiGS-5HacT2HnT gpios */
+#define RB962_WIFI_LED_1	1
+#define RB962_WIFI_LED_2	2
```

Comment:
These GPIOs do not control the WiFi LEDs on my 962UiGS-5HacT2HnT.

Reply:
They are not GPIOs. That's why the defines are "RB962_WIFI" and not "RB962_GPIO". As mentioned in the commit message they are only listed for documentation purpose for if/when someone writes the support code for them...

Reply:
@rmounce does the patchset otherwise work for you? Did it break anything? Can I add a Tested-by? :)

Reply:
You may, I didn't notice anything that broke :)